inst/json/cost_docs.md

```diff
@@ -4,12 +4,10 @@
 
 The first figure shows total costs and cases averted following a 3-year intervention package introduced at year 0.
 
-The red dashed line is showing the maximum global budget determined by the user.
-If no red line is shown the y-axis scale does not intercept this value. The absolute costs are predicted and shown for
-each intervention package. The summary information provided by the user is shown to indicate the ITN usage expected in
+The absolute costs are predicted and shown for each intervention package. The summary information provided by the user is shown to indicate the ITN usage expected in
```

Comment:
Changes to this file are unrelated to the third net type, but were requested in Ellie's original update document.